src/common/util.cc

```diff
@@ -672,4 +672,28 @@ int aeWait(int fd, int mask, uint64_t timeout) {
     return retval;
   }
 }
+
+Status Write(int fd, const std::string &data) {
```

Comment:
I wonder if there is any difference between `SockSend` and this new function.

Reply:
I'm going to create a pr that fixes the socksend function and replaces write with send. Becase send has more flags than write.

Reply:
Do we really use the flag provided by `send`?

Currently `SockSend` has a totally same function body as this function. It is weird.